lib/animina_web/live/profile_photo_live.ex

```diff
@@ -115,7 +115,7 @@ defmodule AniminaWeb.ProfilePhotoLive do
         params = Map.merge(params, file)
         form = Form.validate(socket.assigns.form, params, errors: true)
 
-        Photo.destroy(socket.assigns.current_user.profile_photo)
+        Photo.delete_photo_and_optimized_photos(socket.assigns.current_user.profile_photo)
```

Comment:
This solves the problem on the wrong layer of the application. What if I open the `iex` and delete a user manually? Or a photo resource? We need to solve this in the resource and not in the LiveView layer.